api/src/main/java/com/valtech/aem/saas/api/caconfig/SearchConfiguration.java

```diff
@@ -10,8 +10,11 @@
 
     boolean DEFAULT_ENABLE_BEST_BETS = false;
     boolean DEFAULT_ENABLE_AUTO_SUGGEST = true;
+    boolean DEFAULT_ENABLE_AUTOCOMPLETE = true;
```

Comment:
**Thought**

I would have named it AUTO_COMPLETE or AUTOSUGGEST :D 

Naming variables is one of the hardest problems (aside caching...) in IT haha

Reply:
I agree :)
I've noticed this a while ago, but I will have to change every class, variable name, and it will blow up the changed files set. I'd prefer doing this in another dedicated PR.

Reply:
here is the renaming PR -> https://github.com/valtech-ch/saas-aem-module/pull/129